tests/managed_services/mas_debezium/consumer_pod.py

```diff
@@ -0,0 +1,55 @@
+import shlex
+
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.pod import Pod
+
+
+class ConsumerPod(Pod):
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        teardown=True,
+        privileged_client=None,
+        yaml_file=None,
+        consumer_image=None,
+        kafka_bootstrap_url=None,
+        kafka_sa_client_id=None,
+        kafka_sa_client_secret=None,
+        kafka_test_topic=None,
```

Comment:
these are not used to create the pod; should not be set here but used directly in the command